gdrive/export_client.py

```diff
@@ -169,4 +169,12 @@ def get_qualtrics_response(surveyId: str, responseId: str):
         raise error.ExportError(
             f"No survey response found for responseId: {responseId}"
         )
-    return r.json()
+
+    resp = r.json()
+
+    if resp["status"] != "Complete":
```

Comment:
This check does not belong here. If there is a response, this function should return it.